src/test/java/org/zalando/nakadi/service/EventTypeServiceTest.java

```diff
@@ -215,6 +218,16 @@ public void testFeatureToggleForbidsDeleteEventTypeWithNonDeletableSubscription(
         fail("Should throw AccessDeniedException");
     }
 
+    @Test
+    public void testAllowCreatingEventTypeWithInformationalFieldsFromEffectiveSchema(){
+        final EventType et = EventTypeTestBuilder.builder()
+                .category(EventCategory.DATA)
+                .build();
+        et.setOrderingKeyFields(Collections.singletonList("metadata.occurred_at"));
+        et.setOrderingInstanceIds(Collections.singletonList("metadata.partition"));
+        eventTypeService.create(et, true);
```

Comment:
Shouldn't the test have some validation step?

Reply:
Before the fix, the test will break with exception.

Reply:
Okay I got the point; fixed.